.github/workflows/ci-sharktank.yml

```diff
@@ -26,8 +26,17 @@ jobs:
     name: "Unit Tests and Type Checking"
```

Comment:
Maybe rename to just "Unit Tests" or "Test" so more of the matrix shows up in the UI

![image](https://github.com/user-attachments/assets/1a8f5907-16a3-42ae-b0a4-f8cff89ae1e8)

Could also qualify "torch {{ matrix.torch-version }}"

Here's a mock-up to show where the text gets cut off: 
![image](https://github.com/user-attachments/assets/018b4ffd-d01e-4dc4-ab4c-58c5832be3ab)

Reply:
I renamed to "Unit Tests" for now but we probably want to harmonize this more across different workflows in the future.

Reply:
Nice. Definitely better now. Could still include "torch" next to the torch version but it might be obvious enough to some people.

![image](https://github.com/user-attachments/assets/531df4a1-505d-40df-a7a6-a759be6aa859)

Reply:
Renamed to `"Unit Tests :: ${{ matrix.os }}, ${{ matrix.python-version }}, ${{ matrix.torch-version }}"` as for Windows it was `(windows-2022, 3.11, 2.3.0)` and now have the same order for all jobs. WDYT?

Reply:
Ah, the ordering helps too. I think having two versions right next to each other can be confusing unless you are very familiar with Python _and_ PyTorch versions. I personally have to unpack `3.11, 2.4.1` and think for a second, compared to `3.11, torch 2.4.1`. We don't have many characters to spare in the UI though.

Reply:
Save another character but we should aim to harmonize the naming in a follow up.